### PR TITLE
Bitwise operators in Python

### PR DESCRIPTION
**Pull request by Sin-Sumit**
Both are related to sorted method.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by Sin-Sumit** [comment]
@jdwilkin4 Please review this one.

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,37 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise NOT operation?",
+    Answer: "~",
+    Distractor1: "NOT",
+    Distractor2: "!",
+    Distractor3: "not",
+    Explanation: 
+      "The and keyword is used in Python to perform Bitwise NOT operations.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]: Based on the answer you supplied I think the question should be which operator is used for that bitwise operation.
Let's go with this change

```suggestion
      "In Python, which operator is used to perform Bitwise NOT operations?",
    Answer: "~",
    Distractor1: "++",
    Distractor2: "!",
    Distractor3: "--",
    Explanation: 
      "The ~ operator is used in Python to perform Bitwise NOT operations.",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,37 @@
...
+  },
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise OR operation?",
+    Answer: "|",
+    Distractor1: "OR",
+    Distractor2: "or",
+    Distractor3: "||",
+    Explanation: 
+      "The or keyword is used in Python to perform Bitwise OR operations.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:21]: Based on the answer you supplied I think the question should be which operator is used for that bitwise operation.
Let's go with this change

```suggestion
      "In Python, which operator is used to perform Bitwise OR operations?",
    Answer: "|",
    Distractor1: ">>>",
    Distractor2: "<<<",
    Distractor3: "x",
    Explanation: 
      "The | operator is used in Python to perform Bitwise OR operations.",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,37 @@
...
+  },
+  {
+    Question: 
+      "In Python, what keyword is used to perform Bitwise AND operation?",
+    Answer: "&",
+    Distractor1: "AND",
+    Distractor2: "&&",
+    Distractor3: "&",
+    Explanation: 
+      "The and keyword is used in Python to perform Bitwise AND operations.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:32]: Based on the answer you supplied I think the question should be which operator is used for that bitwise operation.
Let's go with this change

```suggestion
      "In Python, which operator is used to perform Bitwise AND operations?",
    Answer: "&",
    Distractor1: "{{",
    Distractor2: "&&",
    Distractor3: "<",
    Explanation: 
      "The & operator is used in Python to perform Bitwise AND operations.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**